chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertifiedProductDetailsManager.java

```diff
@@ -1,10 +1,14 @@
-package gov.healthit.chpl.manager;
-
-
-import gov.healthit.chpl.dao.EntityRetrievalException;
-import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
-
-public interface CertifiedProductDetailsManager {	
-	public CertifiedProductSearchDetails getCertifiedProductDetails(Long certifiedProductId) 
-			throws EntityRetrievalException;
-}
+package gov.healthit.chpl.manager;
+
+
+import java.util.ArrayList;
+
+import gov.healthit.chpl.dao.EntityRetrievalException;
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+
+public interface CertifiedProductDetailsManager {	
+	public CertifiedProductSearchDetails getCertifiedProductDetails(Long certifiedProductId) 
+			throws EntityRetrievalException;
+
+	ArrayList getIcsFamilyTree(Long certifiedProductId) throws EntityRetrievalException;
```

Comment:
Is this method supposed to have the default access or should it say public/private/protected? Elsewhere in the code we have generic List<Type> or Set<Type> return types rather than the concrete ArrayList and HashSet. You should at least say ArrayList<Type> but to be consistent with everywhere else it should be List<Type>.